AdyenComponents/Boleto/BoletoComponent.swift

```diff
@@ -87,7 +87,15 @@ public final class BoletoComponent: PaymentComponent, LoadingComponent, Presenta
                                       apiContext: apiContext,
                                       fields: formFields,
                                       configuration: configuration,
-                                      onCreatePaymentDetails: { [weak self] in self?.createPaymentDetails() })
+                                      onCreatePaymentDetails: { [weak self] in
+            var paymentMethodDetails: PaymentMethodDetails?
+            do {
+                paymentMethodDetails = try self?.createPaymentDetails()
+            } catch let error {
+                adyenPrint(error)
+            }
+            return paymentMethodDetails
```

Comment:
So in case of error we are going to print message and then use [InstantPaymentDetails(type: paymentMethod.type)](https://github.com/Adyen/adyen-ios/pull/864/files#diff-1b2fc8c8f5c7035f9237a992a77bcb221aba88bf22a1896d8cf5a2bf871c7da1L232)

I guess we need to refactor Boleto component entirely 🤔